lib/messagebird.js

```diff
@@ -226,6 +226,69 @@ module.exports = function (accessKey, timeout) {
       }
     },
 
+    mms: {
+
+      /**
+       * Get a mms message
+       *
+       * @param {String} id
+       * @param {Function} callback
+       * @return {void}
+       */
+      read: function (id, callback) {
+        httpRequest('GET', '/mms/' + id, callback);
+      },
+
+      /**
+       * Send a mms message
+       *
+       * @param {Object} params
+       * @param {Function} callback
+       * @return {void}
+       */
+      create: function (params, callback) {
+        if (params.recipients instanceof Array) {
+          params.recipients = params.recipients.join(',');
```

Comment:
It's better to not alter the incoming params object. User normally expect a library to handle data immutable. What you could do is to create a new object with the params changed in there. 
```suggestion
        var paramsWithJoinedRecipients = Object.assign({}, params);
        if (params.recipients instanceof Array) {
          paramsWithJoinedRecipients.recipients = params.recipients.join(',');
```

Reply:
It seems library supporting node 0.8, 0.10 etc. I think these versions do not support Object.assign. 

Again, You can see the same style all over the codebase. We can discuss.

Reply:
Wow, I wasn't aware that `Object.assign` is ES2015.